aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -96,8 +96,7 @@ static THCTensor* THNN_(newViewWeightMM2d)(THCState *state, THCTensor *weight) {
     int64_t s1 = weight->size(0);
     int64_t s2 = weight->size(1) * weight->size(2) * weight->size(3);
     THCTensor *old_weight = weight;
-    weight = THCTensor_(newWithStorage2d)(state, THTensor_getStoragePtr(weight), weight->storage_offset(),
-                                          s1, -1, s2, -1);
+    weight = THTensor_wrap(weight).view({s1, s2}).unsafeReleaseTensorImpl();
```

Comment:
same

Reply:
see the `view` description above.

Reply:
Excellent, thanks! Makes sense. I had the thought that it might be worth putting this info in a comment, but OTOH maybe that cuts against the idea that we want to excise it as soon as we can...